WebRoot/js/zimbraMail/core/ZmZimbraMail.js

```diff
@@ -2385,17 +2385,20 @@ function(parent, parentElement, adminUrl) {
 
 	if (supportedHelps.indexOf("productHelp") !== -1) {
 		mi = menu.createMenuItem("documentation", {text: ZmMsg.productHelp});
+		mi.getHtmlElement().setAttribute('aria-label', ZmMsg.productHelp + ": " + ZmMsg.link);
```

Comment:
should we merge and create new key? for some languages we might require to change sequence of words which is not possible currently

Reply:
We have intentionally added `aria-label` for NVDA tool with two part as `<option name>: <option type>` such that it will announce `<option name>` and it will pause for 1 second and then it will announce type of that option `<option type>`. As User action dropdown contains few options which are link and as per requirement NVDA tool should announce that option as `Link`